src/components/course/data/tests/hooks.test.jsx

```diff
@@ -507,17 +434,18 @@ describe('useTrackSearchConversionClickHandler', () => {
   };
 
   const mockCourseState = {
-    activeCourseRun: { key: 'course-run-key' },
     algoliaSearchParams: {
       queryId: 'algolia-query-id',
       objectId: 'algolia-object-id',
     },
   };
   const wrapper = ({ children }) => (
     <IntlProvider locale="en">
-      <CourseContext.Provider value={{ state: mockCourseState }}>
-        {children}
-      </CourseContext.Provider>
+      <MemoryRouter initialEntries={['/?queryId=algolia-query-id&objectId=algolia-object-id']}>
```

Comment:
[curious] Is the `MemoryRouter` necessary given the mock `algoliaSearchParams` value is passed into `CourseContext.Provider` directly, without parsing from the page route URL? Same question for the `useLocation` mock below.

Reply:
Nope :) removed 👍🏽